airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresTypingDedupingTest.java

```diff
@@ -72,4 +80,31 @@ protected JdbcCompatibleSourceOperations<?> getSourceOperations() {
     return new PostgresSqlGeneratorIntegrationTest.PostgresSourceOperations();
   }
 
+  @Test
+  public void testMixedCasedSchema() throws Exception {
```

Comment:
this might be worth having in the base test class

Reply:
This I avoided because I don't know how all other destinations behave and might be another whack-a-mole for T+D query changes.